client/core/types.go

```diff
@@ -163,11 +163,13 @@ func (m *Market) marketName() string {
 
 // Exchange represents a single DEX with any number of markets.
 type Exchange struct {
-	Host       string                `json:"host"`
-	Markets    map[string]*Market    `json:"markets"`
-	Assets     map[uint32]*dex.Asset `json:"assets"`
-	FeePending bool                  `json:"feePending"`
-	Connected  bool                  `json:"connected"`
+	Host          string                `json:"host"`
+	Markets       map[string]*Market    `json:"markets"`
+	Assets        map[uint32]*dex.Asset `json:"assets"`
+	FeePending    bool                  `json:"feePending"`
+	Connected     bool                  `json:"connected"`
+	ConfsRequired uint32                `json:"confsrequired"`
+	RegConfirms   *uint32               `json:"confs,omitempty"`
```

Comment:
Only needs to be a pointer in `Exchange` for purposes of `omitempty`, but it's a bit awkward as a pointer in `dexConnection`.  The alternative would be to have -1 or 0 be special values in `dexConnection.regConfirms` that indicate to write `nil` to `Exchange.RegConfirms` in `(*Core).Exchanges`.